runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -150,17 +150,12 @@ public long append(@Nonnull Object object, @Nullable TxResolutionInfo conflictIn
         // The serialization here only serializes the payload because the token is not
         // acquired yet, thus metadata is incomplete. Once a token is acquired, the
         // writer will append the serialized metadata to the buffer.
-        try (ILogData.SerializationHandle sh = ld.getSerializedForm(serializeMetadata)) {
-            int payloadSize;
-            if (skipWriteSizeCheck) {
-                payloadSize = ld.getSizeEstimate();
-                if (log.isTraceEnabled()) {
-                    log.trace("append: payload size is {} bytes.", payloadSize);
-                }
-            } else {
-                payloadSize = ld.checkMaxWriteSize(runtime.getParameters().getMaxWriteSize());
+        try (ILogData.SerializationHandle sh = ld.getSerializedForm(serializeMetadata,
+                !skipWriteSizeCheck, runtime.getParameters().getMaxWriteSize())) {
+            int payloadSize = ld.getSizeEstimate();
```

Comment:
This payloadSize used on L211 as 'write size' is compressed size, but the 'write size' used in the runtime parameter is talking about uncompressed size. I feel like the 'write size' concept is inconsistent in codebase.